package.py

```diff
@@ -0,0 +1,11 @@
+name = "wrap"
+title = "Wrap"
+version = "0.0.3"
+
+client_dir = "ayon_wrap"
+
+plugin_for = ["ayon_server"]
+build_command = ""
+ayon_required_addons = {
+    "core": ">=0.3.0",
```

Comment:
Just realized the addon requires applications addon [client/ayon_wrap/api/lib.py](https://github.com/ynput/ayon-wrap/pull/2/files#diff-91a388b4bbeeab7a779a54c94f01bea2480bacc91b5e55f5caf6a077043b8dc5R3)
```suggestion
    "core": ">=0.3.0",
    "applicatoins": ">=0.2.0",
```